frontend/src/hooks/useGlobalLocalStorage.ts

```diff
@@ -5,6 +5,9 @@ interface IGlobalStore {
     hiddenEnvironments?: Array<string>;
 }
 
+/**
+ * @deprecated use tested `useLocalStorageState` hook instead
+ */
```

Comment:
How many places do we use this today? I migrating off it going to be hard? Also, *why* are we deprecating it? I don't remember any conversation about that.

Reply:
Just this place. I didn't find it anywhere else. We already got rid of this for favorites

Reply:
Safe. Shouldn't be a problem, then. But I still don't understand why we're deprecating it? Does it do something we've moved away from? Does it not work?